modules/logging-enabling-loki-alerts.adoc

```diff
@@ -31,7 +31,7 @@ The `AlertingRule` CR contains a set of specifications and webhook validation de
 
 . Create an `AlertingRule` custom resource (CR):
 +
```

Comment:
The table for tenant application should have something along the lines of 'your application namespace' instead of blank as column 'Valid namespaces for AlertingRule CRs' value.